core/mondoo-azure-security.mql.yaml

```
@@ -283,9 +288,24 @@ queries:
           }
         }
         ```
-  - uid: mondoo-azure-security-rdp-access-restricted-from-internet
-    title: Ensure that RDP access is restricted from the internet
-    impact: 80
+  - uid: mondoo-azure-security-ssh-access-restricted-from-internet-single
+    filters: |
+      asset.platform == "azure-network-security-group"
+      asset.kind == "azure-object"
+    mql: |
+      allNsgTCP= azure.subscription.network.securityGroup.securityRules
+        .where(
+          properties.access == 'Allow'
+            && direction == 'Inbound'
+            && properties.protocol == /TCP|\*/
+            && properties.sourceAddressPrefix == /\*|0\.0\.0\.0|<nw>\/0|\/0|internet|any/
+        )
+      allNsgTCP.all(properties.destinationPortRange != "*")
+      allNsgTCP.none(destinationPortRange.any(fromPort <= 22 && toPort >= 22))
```

@HRouhani Why would this just be `from port == 22 && toPort == 22` ? 
```suggestion
      allNsgTCP.none(destinationPortRange.any(fromPort == 22 && toPort == 22))
```